main/main.c

```diff
@@ -47,6 +47,7 @@
 #include "esp_secure_cert_read.h"
 
 /* Network transport include. */
+#include "freertos/projdefs.h"
```

Comment:
I have never seen this included like this. Why not just include `FreeRTOS.h`?

Reply:
Good catch. This was a mistake.